hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -62,22 +62,35 @@ <h2>Node Status</h2>
        </tr>
     </tbody>
 </table>
-<div>
-    <nav aria-label="...">
-        <ul class="pagination">
-            <li class="page-item" ng-class="{disabled:currentPage==1}"
-                ng-click="handlePagination(currentPage-1,(currentPage==1))">
-                <span class="page-link" tabindex="-1">Previous</span>
-            </li>
-            <li class="page-item active">
-                <span class="page-link">{{currentPage}} </span>
-            </li>
-            <li class="page-item" ng-class="{disabled:lastIndex==currentPage}"
-                ng-click="handlePagination(currentPage+1, (lastIndex==currentPage))">
-                <span class="page-link" tabindex="-1">Next</span>
-            </li>
-        </ul>
-    </nav>
+
+<div class="row">
+    <div class="col-md-6 text-left">
+        <label>Page:</label>
+        <select class="form-select" ng-model="currentPage" ng-change="handlePagination(currentPage, false)">
+            <option ng-repeat="page in getPagesArray()" ng-value="page">{{page}}</option>
+        </select>
+        <span>of {{lastIndex}}. </span>
+        <span ng-if="nodeStatus && nodeStatus.length > 0">
+            Showing {{getCurrentPageFirstItemIndex()}} to {{getCurrentPageLastItemIndex()}} of {{totalItems}} entries.
```

Comment:
I'll leave it up to you if you want to change this, when there are just a few datanodes or showing all the of them in one page (eg. `Showing 1 to 3 of 3 entries.`) I think this is a bit easier to understand. 
```suggestion
            Showing {{getCurrentPageFirstItemIndex()}} to {{getCurrentPageLastItemIndex()}} of the total {{totalItems}} entries.
```

Or we could do this:
```suggestion
            {{getCurrentPageFirstItemIndex()}} - {{getCurrentPageLastItemIndex()}} of {{totalItems}} entries.
```

Reply:
Thanks @dombizita. I think this is enough, I referred to HDFS UI.
<img width="818" alt="image" src="https://github.com/apache/ozone/assets/56783384/97f9c0e3-5dc7-4e12-b57a-08619514a302">